requirements.in

```diff
@@ -0,0 +1,10 @@
+# stdlib backports, no constraints needed
+contextlib2
+
+# external deps
+crcmod>=1,<2
+tornado>=4.2,<5
+
+# tchannel deps
+thriftrw>=0.4,<2
```

Comment:
Also, you'd have to add `requirements.in` to the MANIFEST.in if you want setup.py to read it.